src/Prism.Avalonia/Mvvm/ViewModelLocator.cs

```diff
@@ -21,10 +21,10 @@ static ViewModelLocator()
         /// The AutoWireViewModel attached property.
         /// </summary>
         public static AvaloniaProperty AutoWireViewModelProperty =
-            AvaloniaProperty.RegisterAttached<Control, bool>(
+            AvaloniaProperty.RegisterAttached<Control, bool?>(
                 name: "AutoWireViewModel",
                 ownerType: typeof(ViewModelLocator),
-                defaultValue: false);
+                defaultValue: null);
```

Comment:
The usage of `null` is the preferred method as per, `Prism.WPF`, which gets picked up by `MvvmHelpers.AutowireViewModel(...)`